doc/interface.mld

```diff
@@ -1,40 +1,40 @@
-{0 [odoc] interface guarantees}
+{0 [odoc] Interface Guarantees}
 
 [odoc] has several 'public facing' parts with varying levels of support guarantees.
 This document describes what those interfaces are, what the support levels are
 now, and what we aim for in the future.
 
 {2 Documentation Comments}
 
-The first and most important is the syntax of the documentation comments present in source code.
+The first and most important is the syntax of the documentation comments present in the source code.
 This is relevant to everyone who is writing code that’s intended to be documented by [odoc], so it applies to the widest set of people.
 We have a separate page describing the {{!page-ocamldoc_differences}markup differences from OCamldoc}.
 
 {2 CLI Interface}
 
-The way in which the [odoc] CLI is invoked is not trivial, and it requires careful
-ordering and correct arguments to produce correctly linked documentation. It’s not expected that
-end-users will invoke [odoc] by hand, but rather it will be driven by separate tools. As a consequence, it’s important to preserve the tools’ ability to create good documentation with
-each release of [odoc], so we’ll ensure backward compatibility of the CLI as much as possible.
+The way in which we invoke the [odoc] CLI is not trivial, and it requires careful
+ordering and accuarate arguments to produce correct linked documentation. It’s not expected that
```

Comment:
Not sure if "accurate" is an adjective that applies to CLI arguments, but I think we understand the idea and it avoids the repetition! However, there is a typo:
```suggestion
ordering and accurate arguments to produce correct linked documentation. It’s not expected that
```

"correct link documentation" and "correctly linked documentation" feels different and the latter feels ~correcter~ more correct to me.

Reply:
Nice catch!!